npm-shrinkwrap.json

```diff
@@ -1858,6 +2232,820 @@
       "resolved": "http://registry.npmjs.org/chokidar/-/chokidar-1.6.1.tgz",
       "dev": true,
       "dependencies": {
+        "fsevents": {
```

Comment:
Oops, this sneaked back in. You can remove it by running

    rm -rf node_modules/chokidar/node_modules/fsevents
    npm shrinkwrap

You can also avoid it by making sure you add `--no-optional` to your `npm install` commands. It's a pain, but I'm not sure if there's a better way around it (unless you'd to try switching the project to another NPM client like [yarn](https://github.com/yarnpkg/yarn)?)